dbms/src/Storages/DeltaMerge/DMDecoratorStreams.h

```diff
@@ -24,6 +27,122 @@ namespace DB
 {
 namespace DM
 {
+
+static constexpr size_t UNROLL_BATCH = 64;
```

Comment:
Move the `DB::DM::UNROLL_BATCH` to the scope of `DMDeleteFilterBlockInputStream`? And also move the `DB::DM::UNROLL_BATCH` in `DMVersionFilterBlockInputStream.cpp` to the scope of `DMVersionFilterBlockInputStream`.
I am afraid they may conflict with each other in some cases and bring unexpected behavior.

https://github.com/pingcap/tiflash/blob/1502795b785ac0efea21bc70f6939ac040860482/dbms/src/Storages/DeltaMerge/DMVersionFilterBlockInputStream.cpp#L44-L47